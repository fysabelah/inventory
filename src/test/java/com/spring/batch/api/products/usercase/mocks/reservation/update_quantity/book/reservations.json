[
  {
    "id": "reserva-book-id",
    "sku": "BOOKS-78598-9898",
    "quantity": 2,
    "status": "READY",
    "date": "2024-07-01T08:40:00Z"
  }
]