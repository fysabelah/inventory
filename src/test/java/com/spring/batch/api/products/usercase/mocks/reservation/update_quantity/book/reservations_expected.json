[
  {
    "id": "reserva-book-id",
    "sku": "BOOKS-78598-9898",
    "quantity": 3,
    "status": "READY",
    "date": "2024-07-01T08:40:00Z"
  }
]